nservicebus/azure-storage-queues/native-delayed-delivery.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Native delayed delivery
+component: ASQ
+versions: '[7.1,]'
+tags:
+- Azure
+- Cloud
+- Delayed delivery
+- ASQ
+- Azure Storage Queues
+reviewed: 2016-10-21
+---
+
+[Delayed delivery](/nservicebus/messaging/delayed-delivery) enables sending messages that are delivered at a later time. 
```

Comment:
Hmmm, I'm no native English speaker, but perhaps someone else ( @SeanFeldman ?) could comment on this. Wouldn't "enables sending messages that will be delivered in the near future" (or something like it) sound better? If the current sentence is correct English, just ignore this comment! :)